msg/diagnostic_msg.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+
+uint32[] compute_times # Native code execution time
+uint32[] full_compute_times # Approximate ROS inclusive execution time
+float32 avg_times
```

Comment:
what is the conceptual difference between avg time and full avg times. Is one a moving average? If so should we have the window size as a parameter?